WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/progress/BackupDownloadProgressViewModel.kt

```diff
@@ -100,13 +97,13 @@ class BackupDownloadProgressViewModel @Inject constructor(
     private fun handleState(state: BackupDownloadRequestState) {
         when (state) {
             is NetworkUnavailable -> {
-                _snackbarEvents.postValue(Event(NetworkUnavailableMsg))
+                _errorEvents.postValue(Event(true))
             }
             is RemoteRequestFailure -> {
-                _snackbarEvents.postValue(Event(GenericFailureMsg))
+                _errorEvents.postValue(Event(true))
```

Comment:
> ❓: Do we plan to show different failure reasons (n/w or remote request error) to the user? Currently we always return `Event(true)`, we might want to return actual error or `Unit` instead of `Boolean` if it is never set to `false`.

I've swapped out the boolean for an error type. In commit 82d589e